e2e-tests/cypress/e2e/journeys/insurance/check-your-answers/your-buyer/change-your-answers/change-your-answers-change-company-or-organisation.spec.js

```diff
@@ -174,7 +173,6 @@ context('Insurance - Check your answers - Company or organisation - Your buyer p
 
         // as html, cannot use checkText so checking contains following fields
         row.value().contains(fieldVariables.newValueInput);
-        row.value().contains(application.BUYER[COUNTRY]);
```

Comment:
Whlie you're in here could you fix typo in this E2E test and some others?

```js
Your buyer page-
```

Missing a space